src/Simplex/Chat/Types/Shared.hs

```diff
@@ -51,11 +51,6 @@ instance ToJSON GroupMemberRole where
 
 data GroupAcceptance = GAAccepted | GAPending deriving (Eq, Show)
```

Comment:
not clear what happened here

Reply:
if this comment refers to deletion of instances - they were not used (initially this field were added to db, then removed from db in same pr, and also their encoding was wrong for db - mismatched text/blob)